app/models/repository.rb

```diff
@@ -0,0 +1,2 @@
+class Repository < ActiveRecord::Base
```

Comment:
@bturner-r7 and I agree that creating a model that isn't used isn't fantastic. If we don't end up adding support for hooks in this PR we need to remove the `form_for` usage in the repository form and use `form_tag` instead.